packages/core/src/discovery.js

```diff
@@ -95,6 +98,45 @@ function parseDomResources({ url, domSnapshot }) {
   }, new Map([[rootResource.url, rootResource]]));
 }
 
+async function parseImageSrcset(snapshot, discovery) {
+  let log = logger('core:snapshot');
```

Comment:
shouldnt logger be discovery ?

Reply:
All logger in this file uses `core:snapshot`